client-v2/src/actions/__tests__/Persistence.spec.ts

```diff
@@ -0,0 +1,208 @@
+import { createStore, compose, applyMiddleware } from 'redux';
```

Comment:
This test covers a good surface area, nice to see 👍

Reply:
I wonder whether I should go through and test more actions that might exhibit this behaviour ...